evutil.c

```diff
@@ -372,6 +372,13 @@ evutil_make_listen_socket_reuseable(evutil_socket_t sock)
 }
 
 int
+evutil_socket_bind_address(evutil_socket_t sock, const struct sockaddr *sa,
+    int socklen)
+{
+	return bind(sock, sa, socklen);
```

Comment:
AFAICS this is not the common practice "adding wrapper, just in case", IOW only if do some `#ifdef` magic or some other quirks.
And if you read the code then you see "just bind()", and know what it does.